ndk/src/relay/pool/index.test.ts

```diff
@@ -1,4 +1,4 @@
-import "websocket-polyfill";
```

Comment:
Why is this removed here as part of this PR?

Reply:
oops, it just a mistake

Reply:
Want to give this a rebase while you're at it?

Reply:
I've readd it